src/dialects/sqlite/connection-manager.ts

```diff
@@ -87,6 +87,7 @@ export class SqliteConnectionManager extends AbstractConnectionManager<SqliteCon
 
       this.connections.set(connectionCacheKey, connectionInstance);
     });
+    this.sequelize.options.databaseVersion = await this.sequelize.fetchDatabaseVersion();
```

Comment:
This is actually breaking the integration tests!

Reply:
The way you wrote it caused an infinite loop since `fetchDatabaseVersion` executes a query which calls `getConnection` which calls `fetchDatabaseVersion`

That's why `initDatabaseVersion` in `AbstractConnectionManager` uses a hack to force `fetchDatabaseVersion` to use a specific connection

The best solution would be to make `_initDatabaseVersion` protected instead of hard private, and call it from `SqliteConnectionManager`